### PR TITLE
Initial rebrand work

### PR DESCRIPTION
**Pull request by davidallardsb**
#### What does this PR do? (please provide any background)
Updates the Capital Breaks brand to be encore tickets

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/capital-breaks/src/less/variables.less [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 //== Scaffolding
 //
-@body-bg:                   #fafafa;
-@text-color:                rgba(0,0,0,0.8);
+@body-bg:                   #f1f5f7;
```
Comment (nickine) [inline brands/capital-breaks/src/less/variables.less]: Can you replace with from colours.less? `@gray-lighter`

File: brands/capital-breaks/src/less/variables.less [hunk 2]
```diff
@@ -13,8 +13,8 @@
...
+@text-color:                #1B1B1B;
```
Comment (nickine) [inline brands/capital-breaks/src/less/variables.less]: `@gray-darker`

File: brands/capital-breaks/src/less/colours.less [hunk 3]
```diff
@@ -4,22 +4,22 @@
...
-@brand-primary:   rgb(255,208,59);
-@brand-secondary: @logo-primary;
-@brand-success:   rgb(125,157,59);
-@brand-warning:   rgb(240,173,78);
-@brand-danger:    rgb(216,43,38);
-@brand-info:      rgb(91,199,222);
+@brand-primary:   #4AC3F4;
+@brand-secondary: #F01553;
+@brand-success:   #1FA91F;
+@brand-warning:   #F01553;
```
Comment (nickine) [inline brands/capital-breaks/src/less/colours.less]: Can we use a different colour to distinguish between `secondary` and `warning`?

File: brands/capital-breaks/src/less/colours.less [hunk 4]
```diff
@@ -4,22 +4,22 @@
...
+@brand-danger:    #D61725;
+@brand-info:      #4AC3F4;
```
Comment (nickine) [inline brands/capital-breaks/src/less/colours.less]: Same as comment for `warning`

File: brands/capital-breaks/src/less/colours.less (hunk 3 above)
Comment (davidallardsb) [reply brands/capital-breaks/src/less/colours.less]: The Encore Brand has these colours the same when we consider how the-wall is configured. I have updated the naming to make it more apparent. I think Jodie will probably change this though as I am just stepping in whilst he is on Hol :)

**Comment by nickine** [comment]
Approved for MVP, @jodiedoubleday to review post golive/when back in the office.

**Pull request merged**